### ISSUE TITLE
Can not import generated query class

### ISSUE THREAD

**Issue by m7mdra** [apollographql/apollo-android#3351]
**Summary**
Im using version 3 of this library and for some reasons im not able to import generated query class when they exists in build directory

**Version**
version 3 beta 4

After i migrated from version 2 to 3 and followed the migration guide in the documentaion i wasnt able to call query class like before

here is my module level `gradle.build` 
```groovy

plugins {
    id 'com.android.application'
    id 'kotlin-android'
    id 'kotlin-android-extensions'
    id 'com.apollographql.apollo3'
    id 'kotlin-kapt'
}
apollo {
    packageName = "com.example"
    srcDir("src/main/graphql/com/example")
    customScalarsMapping = [
            "Date": "java.util.Date"
    ]
}
android {
    compileSdkVersion 31
    buildToolsVersion "30.0.3"

    defaultConfig {
        ..
        ..
    }

    buildTypes {
        release {
            minifyEnabled true

            ndk {
                abiFilters "arm64-v8a", "armeabi-v7a"
            }

            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
    kotlinOptions {
        jvmTarget = '1.8'
    }
}
def koin_version = "3.1.2"
dependencies {
    
    implementation("com.apollographql.apollo3:apollo-runtime:+")
    implementation("com.apollographql.apollo3:apollo-api:+")
    implementation 'com.apollographql.apollo3:apollo-rx3-support:+'
    implementation("com.apollographql.apollo3:apollo-adapters:+")

}
kapt {
    correctErrorTypes true
}
```
`build.gradle` project level

```groovy
buildscript {
    ext.kotlin_version = "1.5.20"
    repositories {
        google()
        mavenCentral()
        jcenter()
        maven {
            url "https://jitpack.io"
        }

    }
    dependencies {
        classpath 'com.android.tools.build:gradle:4.2.2'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
        classpath("com.apollographql.apollo3:apollo-gradle-plugin:+")
    }
}

allprojects {
    repositories {
        google()
        mavenCentral()
        jcenter()
        maven {
            url "https://jitpack.io"
        }

    }
}
```

Android Studio 4.2.1
Gradle 6.7.1

Thanks in advance.

**Comment by m7mdra** [comment]
@martinbonnin 

i can generate them just fine but im not able to build since the imports are not resolved, thanks for fast replay

<img src="https://user-images.githubusercontent.com/16242735/133263777-1dedd7d2-7010-45ae-acf9-e5c4bb6d7b67.png" height="50%"/>

**Comment by m7mdra** [comment]
here is a minimal project to reproduce the bug [Here](https://github.com/m7mdra/Apollo3Test)

**Comment by martinbonnin** [comment]
Thanks for the reproducer, I can reproduce 👍 . This seems to be an Android Studio-only issue. Building the app from the command line doesn't show the error. I'll dig into this.

**Comment by m7mdra** [comment]
Thanks for the follow up, i will try to use different version , arctic fox maybe and check if the problem is still persistent or i will succumb to use cmd , thanks again.

will close this since its not related to library.

**Comment by martinbonnin** [comment]
It should be fixed with `alpha06` that is just out (See #3354). Let me know how that works
